### ISSUE TITLE
Feature Request: In-Game Cheat Menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#129]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On which system and loader?


Please provide any additional information below.


```

Original issue reported on code.google.com by `findonov...@gmail.com` on 3 Oct 2009 at 9:36
- Merged into: #84

**Comment by GoogleCodeExporter** [comment]
```
This issue should be merged into issue #84
```

Original comment by `Gamewiza...@gmail.com` on 4 Oct 2009 at 4:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `emuki...@gmail.com` on 27 Oct 2009 at 2:08
- Changed state: **Duplicate**
